docs/docs/generating-nlu-data.mdx

```diff
@@ -30,14 +30,22 @@ Conversation-Driven Development (CDD) means letting real user conversations guid
 development. For building a great NLU model, this means two key things:
 
 ### Gather Real Data
```

Comment:
The content here is great, but shows up as a huge wall of text -- i'd recommend listing the 2 reasons in a list or splitting up the paragraphs even more; usually we stick to 2-3 sentences to make the reading less intimidating